src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -786,7 +810,8 @@ mod tests {
                     .unwrap(),
             )
             .unwrap();
-            let uds_path = format!("test_vsock_{}.sock", name);
+
+            let uds_path = get_available_file(format!("test_vsock_{}.sock", name));
```

Comment:
Can we use `Tempfile` instead to generate a unique path?

Reply:
Yep, used it. 

See latest update, please.